hazelcast/proxy/set.py

```diff
@@ -22,6 +22,7 @@ class Set(PartitionSpecificProxy):
     """Concurrent, distributed implementation of Set"""
 
     def add(self, item):
+        # type: (Any) -> bool
```

Comment:
The return types are not correct, we return a `Future`, maybe you can use `Future[bool]` here?